docs/source/usage.rst

```diff
@@ -457,6 +458,19 @@ You can take advantage of nested interpolations to perform custom operations ove
     >>> c.a_plus_b
     3
 
+
+More advanced resolver naming features include the ability to prefix a resolver name with a
+namespace, and to use interpolations in the name itself. The following example demonstrates both:
+
+.. doctest::
+
+    >>> OmegaConf.register_new_resolver("mylib.plus1", lambda x: x + 1)
+    >>> c = OmegaConf.create({"func": "plus1",
+    ...                       "x": "${mylib.${func}:3}"})
```

Comment:
Use Black to determine formatting of code snippets in the docs:

```python
c = OmegaConf.create({"func": "plus1", "x": "${mylib.${func}:3}"})
```
or
```python
c = OmegaConf.create(
    {
        "func": "plus1",
        "x": "${mylib.${func}:3}",
    }
)

```

I prefer the second. your call.

Reply:
Done in 18ec4cb (went for second one because first one went slightly over the doc width limit, forcing scrolling)